spec/rubocop/cop/sequel/irreversible_migration_spec.rb

```diff
@@ -73,4 +81,17 @@
       expect(offenses).to be_empty
     end
   end
+
+  context 'when a change block is used outside of a Sequel migration' do
+    let(:source) do
+      <<~SOURCE
+        it { expect { subject }.to change { document_count(user_id) }.by(-1) }
```

Comment:
Test case taken directly from the reported issue:
https://github.com/rubocop/rubocop-sequel/issues/44